features/situation-feedback/api/src/main/java/org/opennms/features/situationfeedback/api/AlarmFeedback.java

```diff
@@ -156,10 +179,18 @@ public FeedbackType getFeedbackType() {
         return feedbackType;
     }
 
+    public Boolean getRootCause() {
+        return rootCause;
+    }
+
     public String getReason() {
         return reason;
     }
 
+    public List<String> getTags() {
+        return tags;
```

Comment:
Change to `unmodifiableSet` if we switch to a set.
```suggestion
        return Collections.unmodifiableList(tags);
```

Reply:
Let's still update this to the unmodifiable list to maintain immutability in this class.